doc/09-object-types.md

```diff
@@ -1470,6 +1470,11 @@ Cleanup Items:
   servicechecks\_age              | Duration              | **Optional.** Max age for servicechecks table rows (start\_time). Defaults to 0 (never).
   systemcommands\_age             | Duration              | **Optional.** Max age for systemcommands table rows (start\_time). Defaults to 0 (never).
 
+> **Supported units**
+>
+> Supported suffixes include ms (milliseconds), s (seconds), m (minutes), h (hours) and d (days).
+> Check the [language-reference](17-language-reference.md#numeric-literals-)
```

Comment:
I would use "language reference" (space instead of dash) in the link text to make it consistent with the title of that page and add a period at the end of the sentence.

Reply:
chanced to the proposed "language reference"